core/arch/arm/mm/core_mmu.c

```diff
@@ -216,10 +217,17 @@ void core_init_mmu_map(void)
 	/* map what needs to be mapped (non-null size and non INTRAM/EXTRAM) */
 	map = in;
 	while (map->type != MEM_AREA_NOTYPE) {
-		if (map->va)
-			panic();
-
-		map->va = map->pa;	/* 1-to-1 pa = va mapping */
+		if (map->va) {
+#ifdef CFG_WITH_PAGER
+			if (map->va & SMALL_PAGE_SIZE)
+				panic();
+#else
+			if (map->va & (1 << 20))
```

Comment:
`1 << 20` as a constant (using `SECTION_SHIFT` or `SECTION_SIZE` as in `core_mmu_v7.c`